bezier-rs/docs/interactive-docs/src/components/SliderExample.vue

```diff
@@ -38,7 +38,9 @@ export default defineComponent({
 	},
 	data() {
 		const sliders: SliderOption[] = this.templateOptions.sliders;
-		return Object.assign({}, ...sliders.map((s) => ({ [s.variable]: s.default })));
+		return {
+			sliderData: Object.assign({}, ...sliders.map((s) => ({ [s.variable]: s.default }))),
```

Comment:
Instead of `Object.assign({}, ...X}`, can't you just do `{ ...X }`?

Reply:
The spread operator in your solution unpacks the list, not the maps therein. `sliders = [{variable: 't1', default: 0.5}, {variable: 't2', default: 0.2}]` produces `{0: {t1: 0.5}, 1: {t2: 0.2}`, but we want `{t1: 0.5, t2: 0.2}`